packages/google_sign_in/google_sign_in_web/pubspec.yaml

```diff
@@ -22,9 +22,14 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  google_sign_in_platform_interface: ^2.0.0
+  google_sign_in_platform_interface: ^2.2.0
   js: ^0.6.3
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
+
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides:
+  google_sign_in_platform_interface:
+    path: ../../google_sign_in/google_sign_in_platform_interface
```

Comment:
(This indeed needs to go.)

Reply:
I'm just waiting until the platform interface PR has landed to remove overrides :)